lib/src/executables/dart2_upgrade.dart

```diff
@@ -36,6 +37,13 @@ import '../ignoreable.dart';
 import '../util.dart';
 
 const _backwardsCompatFlag = '--backwards-compat';
+const _helpFlag = '--help';
+const _changesRequiredOutput = """
+To update your code, switch to Dart 2.1.0 and run the following commands:
+  pub global activate over_react_codemod ^<version>
```

Comment:
Let's put the current version in here to start and I'll update our release event configuration so that it gets updated automatically on every release PR.
```suggestion
  pub global activate over_react_codemod ^1.0.1
```